cli/azd/cmd/root.go

```diff
@@ -23,7 +23,7 @@ import (
 	"go.opentelemetry.io/otel/codes"
 )
 
-func NewRootCmd() *cobra.Command {
+func NewRootCmd(platformAgnosticHelp bool) *cobra.Command {
 	prevDir := ""
 	opts := &internal.GlobalCommandOptions{}
 
```

Comment:
I think we could simply put this flag on `GlobalCommandOptions` with documentation on how it works.